Remotion/Web/ClientScript/Scripts/TreeView.ts

```diff
@@ -187,8 +189,13 @@ class WebTreeView
           let hasChildren = activeTreeNode.getAttribute ('aria-expanded') === 'true';
           if (hasChildren)
           {
-            const newTreeNode = activeTreeNode.querySelector<HTMLElement> (':scope > ul > li')!;
-            WebTreeView.SetFocus (newTreeNode, activeTreeNode);
+            const nodeIndex = treeNodes.indexOf(activeTreeNode);
+            const liNode = parentTreeNodeListItems[nodeIndex];
+            const newListItemChildNode = liNode.querySelector<HTMLElement> (':scope > ul > li:first-child')!;
+
+            const childNodeIndex = parentTreeNodeListItems.indexOf(newListItemChildNode);
+            const newNode = treeNodes[childNodeIndex];
+            WebTreeView.SetFocus (newNode, activeTreeNode);
```

Comment:
naming: 
nodeIndex -> activeTreeNodeIndex
liNode -> activeTreeNodeListItem
newListItemChildNode -> newTreeNodeListItem

logic: except for the fact that we already computed the relationship between the LI and the Anchor tag, there's no reason for us to not simply expand n the CSS-selector and use it to pick the correct anchor.